lib/runnable.js

```diff
@@ -450,3 +452,20 @@ Runnable.prototype._timeoutError = function(ms) {
   }
   return new Error(msg);
 };
+
+/**
+ * Generate an undefined error if `err` is not defined.
+ *
+ * @param {*} [err] - Hopefully an Error
+ * @return {Error}
+ */
+
+Runnable.getError = function(err) {
```

Comment:
This doesn't appear to be used anywhere but this file. Why export it? Seems to unnecessarily complicate calling it (L358,L381) for no benefit.